ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -210,17 +211,42 @@ handle('getDesktopVersion', async () => ({ desktopVersion: releaseVersion }));
  */
 handle('cacheDaemonStatus', async () => await cacheDaemonManager.status());
 
+/**
+ * Returns the current log level
+ */
+handle('getLogLevel', () => store.get('logLevel'));
+
+/**
+ * Sets the log level
+ */
+handle('setLogLevel', (logLevel) => store.set('logLevel', logLevel));
+
+/**
+ * Returns the path to the log file
+ */
+handle('getLogPath', () => {
+  switch (os.platform()) {
+    case 'win32':
+      return `${
+        process.env.USERPROFILE ?? '%USERPROFILE%'
+      }\\AppData\\Roaming\\Boundary\\logs\\desktop-client.log`;
```

Comment:
why are we using double slashes here?

Reply:
It's just escaping the backslashes, so the user only sees the one backslash, I have an example of what it looks like on windows in the PR description